doc/kor/utility_token.md

```diff
@@ -0,0 +1,226 @@
+# ISKRA Utility Token(ERC20) Contract
+
+## 개요
+- `contracts/token/ERC20/UtilityToken.sol` 파일은 ERC20 표준을 따르는 샘플 토큰 구현체로서, 이름(name), 심볼(symbol), 최초 minter를 배포 때 정할 수 있도록 구현된 토큰입니다.
+- 배포시 초기 발행량은 0이며, minter 권한이 있는 account로 mint나 burn을 할 수 있습니다.
+- minter는 무제한 발행이 가능합니다.
+- owner는 minter를 추가/삭제할 수 있습니다.
+- decimal은 18로 고정입니다.
+- 초기 배포되면 owner가 총 통화량을 모두 소유하게 됩니다.
```

Comment:
Based on the description above, a UtilityToken must have an initialSupply of 0.
However, here it says that there is a supply owned by owner.